TravelerKit/Models/Price.swift

```diff
@@ -13,6 +13,17 @@ public struct Price: Decodable {
     public let currency: String
 
     public var localizedDescription: String? {
-        return value.priceDescription(currencyCode: currency)
+        let numberFormatter = NumberFormatter()
+        numberFormatter.locale = Locale.current
+        numberFormatter.numberStyle = .currency
+        numberFormatter.currencySymbol = getSymbol()
+        
+        return numberFormatter.string(for: self.value)
+    }
+    
+    func getSymbol() -> String? {
```

Comment:
Swift doesnt have `get` in front of method names.

also the method is unnecessary, its not being reused anywhere